fastlane/lib/fastlane/swift_lane_manager.rb

```diff
@@ -6,7 +6,7 @@ class SwiftLaneManager < LaneManagerBase
     # @param lane_name The name of the lane to execute
     # @param parameters [Hash] The parameters passed from the command line to the lane
     # @param env Dot Env Information
-    def self.cruise_lane(lane, parameters = nil, env = nil, disable_runner_upgrades: false)
+    def self.cruise_lane(lane, parameters = nil, env = nil, disable_runner_upgrades: false, swift_server_port: 2000)
```

Comment:
Ah, my bad! For some reason I thought if a `nil` was passed in that it would still default to `2000`. Let's change this too...

```rb
def self.cruise_lane(lane, parameters = nil, env = nil, disable_runner_upgrades: false, swift_server_port: nil)
  swift_server_port ||= 2000
```

This will clean up `command_line_handler.rb` (see other comment)